### PR TITLE
PAPAY-4244

### PR DESCRIPTION
**Pull request by anatoliikovalenko**

### CODE REVIEW

File: stable/proxysql/templates/deployment.yaml [hunk 1]
```diff
@@ -0,0 +1,57 @@
+{{- if .Values.enable }}
```
Comment (kahootali) [inline stable/proxysql/templates/deployment.yaml]: I dont think the enabled check should be added here... it should be added when adding it as dependency

Comment (anatoliikovalenko) [reply stable/proxysql/templates/deployment.yaml]: I don't get it. Could you explain?

Comment (kahootali) [reply stable/proxysql/templates/deployment.yaml]: I mean these checks should be removed from here, and when adding this chart as a dependency we can have a conditional check like
```
dependencies:
  - name: subchart1
    condition: subchart1.enabled
```
this way this chart wont even be downloaded into the dependencies

Comment (anatoliikovalenko) [reply stable/proxysql/templates/deployment.yaml]: `{{- if .Values.enable }}` removed

**Review by kahootali — APPROVED** [review]

**Pull request merged**